content/en/blog/2023/ambient-performance-savings/index.md

```diff
@@ -0,0 +1,132 @@
+
+{{< image width="100%"
+    link="savings.png"
+    caption="Total Memory and CPU: Consumption and Allocation"
+    >}}
+
+Looking at [total CPU and memory consumption numbers](#a-closer-look), we see that we can remove the allocations required for a sidecar container in every application pod in the mesh. The result is that memory usage of Istio’s dataplane in the ztunnel-only ambient scenario sees *more than 99% savings* compared to sidecar scenarios, and still 89% savings when waypoints are added. Looking at CPU, ztunnel once again sees 82% savings compared to the sidecar scenario, and 40% savings when waypoints are deployed.
+
+## Where the Savings Are Coming From
+
+Ambient mesh was designed to minimize resource requirements for users in their Kubernetes clusters. To explain how ambient does this, we must first clarify _allocation_ versus _utilization_. When deploying a Kubernetes cluster, nodes determine the overall capacity of the cluster and customer deployments, and pods are an allocation of that capacity. Utilization is a measure of how well this is done. As an architecture, sidecars interfere with effective utilization as they:
+
+- Define a high minimum for allocation at any scale
+- Can strand capacity by reserving more than is needed
+- Shift operational burden to adjust allocation if more than the default is needed
+
+The sidecar architecture forces this mode of allocation on users. Ambient solves these issues by leveraging a new architecture that separates the responsibilities of zero-trust networking and L7 policy handling. This is done with two new components to Istio: ztunnels and waypoint proxies.
+
+- [Ztunnels](/blog/2023/rust-based-ztunnel/) are a brand new Istio component written in Rust that are designed to be fast, secure, and lightweight. Ztunnels are deployed per node and serve all pods co-located on the node. They enable the most basic service mesh configurations for L4 features such as mTLS, telemetry, authentication, and L4 authorizations.
+- [Waypoint proxies](/blog/2023/waypoint-proxy-made-simple/) provide L7 mesh features such as VirtualService routing, L7 telemetry, and L7 authorizations policies. Waypoints are still based on Envoy and are deployed per namespace or service account.
+
+These ztunnels and waypoint proxies work in tandem to replace sidecars in the Istio service mesh, and most importantly, unlike sidecars they scale independent to a user's applications pods. So let’s take a closer look at how the two architectures compare in the scenarios above.
+
+## A Closer Look
+
+{{< image width="100%"
+    link="cpu-by-pod.png"
+    caption="CPU usage by pod for sidecar and ambient pods"
+    >}}
+
+{{< image width="100%"
+    link="memory-by-pod.png"
+    caption="Memory usage by pod for sidecar and ambient pods"
+    >}}
+
+{{< image width="100%"
+    link="resource-usage.png"
```

Comment:
Is it possible the convert this png into an editable table?

Reply:
sorry but this blog is not ready yet.  we  can look into an editable table.